Index: json/elements.inl
===================================================================
--- json/elements.inl	(revision 4038)
+++ json/elements.inl	(revision 4068)
@@ -68,12 +68,12 @@
 
 class UnknownElement::ConstCastVisitor : public ConstVisitor
 {
-   virtual void Visit(const Array& array) {}
-   virtual void Visit(const Object& object) {}
-   virtual void Visit(const Number& number) {}
-   virtual void Visit(const String& string) {}
-   virtual void Visit(const Boolean& boolean) {}
-   virtual void Visit(const Null& null) {}
+   virtual void Visit(const Array& /*array*/) {}
+   virtual void Visit(const Object& /*object*/) {}
+   virtual void Visit(const Number& /*number*/) {}
+   virtual void Visit(const String& /*string*/) {}
+   virtual void Visit(const Boolean& /*boolean*/) {}
+   virtual void Visit(const Null& /*null*/) {}
 };
 
 template <typename ElementTypeT>
@@ -88,12 +88,12 @@
 
 class UnknownElement::CastVisitor : public Visitor
 {
-   virtual void Visit(Array& array) {}
-   virtual void Visit(Object& object) {}
-   virtual void Visit(Number& number) {}
-   virtual void Visit(String& string) {}
-   virtual void Visit(Boolean& boolean) {}
-   virtual void Visit(Null& null) {}
+   virtual void Visit(Array& /*array*/) {}
+   virtual void Visit(Object& /*object*/) {}
+   virtual void Visit(Number& /*number*/) {}
+   virtual void Visit(String& /*string*/) {}
+   virtual void Visit(Boolean& /*boolean*/) {}
+   virtual void Visit(Null& /*null*/) {}
 };
 
 template <typename ElementTypeT>
@@ -402,7 +402,7 @@
 //////////////////
 // Null members
 
-inline bool Null::operator == (const Null& trivial) const
+inline bool Null::operator == (const Null& /*trivial*/) const
 {
    return true;
 }
