--- FindwxWidgets.cmake.org	2017-08-23 15:37:34.715190600 +0200
+++ FindwxWidgets.cmake	2017-08-23 15:42:52.501658500 +0200
@@ -498,9 +498,9 @@ if(wxWidgets_FIND_STYLE STREQUAL "win32")
       set(_WX_TOOL gcc)
     elseif(MSVC)
       set(_WX_TOOL vc)
-      if(MSVC_VERSION EQUAL 1910)
+      if(MSVC_VERSION GREATER_EQUAL 1910)
         set(_WX_TOOLVER 141)
-      elseif(MSVC_VERSION EQUAL 1900)
+      elseif(MSVC_VERSION GREATER_EQUAL 1900)
         set(_WX_TOOLVER 140)
       elseif(MSVC_VERSION EQUAL 1800)
         set(_WX_TOOLVER 120)
